content/deyemiobaa/building-a-todolist-with-rails.md

```diff
@@ -0,0 +1,354 @@
+---
+Title: "Bulding a ToDo List with Ruby on Rails"
+Description: "Learn how to build a ToDo List with Ruby on Rails"
+DatePublished: "2022-04-28"
+Categories:
+  - "computer-science"
+  - "web-development"
+Tags:
+  - "Ruby on Rails"
+  - "Ruby"
+  - "PostgreSQL"
+  - "Database"
+CatalogContent:
+  - "learn-ruby"
+  - "learn-rails"
+---
+
+_**Prerequisites:** Ruby, Ruby on Rails, PostgreSQL_
+
+_**Versions:** Ruby 3.2.0, Ruby on Rails 7.0.4, PostgreSQL 15_
+
+## Introduction
+
+This article is an introduction to building a full-stack app using Ruby on Rails. For this article, we are building a To-do list.
+
+I was inspired to write this article while learning Ruby on Rails. I often got lost looking for information on the official Rails guide and I want to help you reduce the time it’ll take you to get started with building your own Rails app.
+
+The article is beginner friendly, and I hope it helps you learn something new.
+
+## What I’ll talk about:
+
+1. Setting up a new Rails app.
+2. Understanding the MVC design pattern.
+3. Creating a new schema in your local database using migrations.
+4. Handling requests within controller methods.
+5. Using ERB to write logic and display content in HTML.
+6. Understanding the Rails routing system.
+
```

Comment:
[Nitpick] Maybe you can list these are the main points of your article(e.g., "This article will...").  
```suggestion
## What I’ll talk about:

1. Setting up a new Rails app.
2. Understanding the MVC design pattern.
3. Creating a new schema in your local database using migrations.
4. Handling requests within controller methods.
5. Using ERB to write logic and display content in HTML.
6. Understanding the Rails routing system.

```